### DISCUSSION TITLE
battery metrics for ups (APC-EATON-NUT)

### DISCUSSION THREAD

**Discussion by tradexsrl** [netdata/netdata#17903] [Q&A] [answered]
hi , i'm not able to find a metrics related to the battery end of life.
i have installed netdata agent and activated the conf to monitor apc ups with the goal to gets notified when the battery end of life is approaching so i need to call technician to replace it.
please let me know

**Comment by ilyam8** [comment, marked as answer]
Netdata monitors [status](https://man.archlinux.org/man/apcaccess.8.en#:~:text=apcupsd%20was%20started-,STATUS,-%3A%20UPS%20status.%20One) of APC UPSes and has this alarm:

https://github.com/netdata/netdata/blob/8271db8c6e096ddc3df58037207722db38fa08df/src/health/health.d/apcupsd.conf#L93-L101

**Comment by tradexsrl** [comment]
many many thank's

**Comment by tradexsrl** [comment]
only one question: how many days before battery eol this event fire up?
because is not clear

**Reply by ilyam8** [reply]
This is the UPS status. You can probably find this in the documentation.

**Reply by tradexsrl** [reply]
sorry but i'm confused now....your reply point to apcupsd.status  which means "info: APC UPS battery has reached the end of its lifespan and needs to be replaced" 
what i understand is this is a metric that netdata collect but when fire up ? only the day when the ups report that lifespan is terminated?
normally this fireup within 30 days or before. i expect to see a push notification or an email notification like in whoisquery 30 days and 15 days before a domain expiry

**Reply by ilyam8** [reply]
I don't know, we're just monitoring the state of the UPS - I don't know when the UPS will _decide_ to change its status to "REPLACEBATT". I'm guessing you can find this information in your UPS documentation.

UPS doesn't expose the number of seconds until the end of battery life as a metric. So we can just monitor its status and notify when the UPS changes its status to REPLACEBATT.

**Reply by tradexsrl** [reply]
ok understand...shame i will need to install another vm from APC used to monitor the ups infrastructure
